pkg/operators/ebpf/ebpf.go

```diff
@@ -580,7 +588,29 @@ func (i *ebpfInstance) Start(gadgetCtx operators.GadgetContext) error {
 		if !p.fromEbpf {
 			continue
 		}
-		constReplacements[name] = paramMap[name].AsAny()
+		paramValue := paramMap[name].AsAny()
+
+		switch p.TypeHint {
+		case api.TypeIP:
+			i.logger.Infof("handling IP param: %q", name)
+			ipAddr := ipAddr{}
+			ipParam := paramValue.(net.IP)
+			if ip := ipParam.To4(); ip != nil {
+				ipAddr.version = 4
+				copy(ipAddr.v6[:4], ip)
+			} else if ip := ipParam.To16(); ip != nil {
+				copy(ipAddr.v6[:], ip)
+				ipAddr.version = 6
+			} else {
+				// TODO: is it possible?
```

Comment:
please handle this TODO

Reply:
Yup, it is possible, this case will run when `ipParam.To4()` or `ipParam.To6()` throws an error. So I believe we just have to remove the comment